corehq/util/datadog/utils.py

```diff
@@ -87,3 +107,8 @@ def get_url_group(url):
         return parts[3] if len(parts) >= 4 else default
 
     return default
+
+
+def notify_datadog(metrics= {}):
```

Comment:
i am looking forward to make this a more generic method and not bound to just `gauge`. Though would definitely update this to `update_datadog_metrics`.